roles/nfd_undeploy/tasks/main.yml

```diff
@@ -18,3 +18,8 @@
 
 - name: Delete the namespace of the NFD Operator
   command: oc --ignore-not-found=true delete -f "{{ nfd_operator_namespace }}"
+
+- name: Delete labels created by the NFD Operator
```

Comment:
Isn't that something the operator should do when it's deleted?

Reply:
no, labels are not deleted when the operator is gone